index.html

```diff
@@ -843,22 +843,10 @@ <h2>Newsletter</h2>
         <div class="section-header">
```

Comment:
We must remove the whole section (or just comment out) and also from menu.

Reply:
I think we should keep it until we have the photos available. Then we replace the whole section, with the event photos.

Reply:
Makes sense.
But have the `resgister` button and section with nothing bellow is weird.

For now the form says inscription is over, so won't be a problem to still show it.